### PR TITLE
feat(dedicated-cloud): adding active node in datastore page

### PR DESCRIPTION
**Pull request by carsso**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | MANAGER-14993
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [x] Breaking change is mentioned in relevant commits

## Description

Following some incidents, it would be nice allow customers to see on which node their datastores are.

Capture d’écran 2024-07-10 à 23 48 29

Complementary to #12257

At the same time, keeping a single profile field (field "fullProfile" from the API, the same in use in the vSphere plugin UI) to avoid "duplicated/confusing" information in the datastore page that looked like this :
Capture d’écran 2024-07-10 à 23 52 14

## Related

### CODE REVIEW

File: packages/manager/apps/dedicated/client/app/components/dedicated-cloud/datacenter/datastore/dedicatedCloud-datacenter-datastore.html [hunk 1]
```diff
@@ -34,6 +29,17 @@ <h2 data-translate="dedicatedCloud_tab_datastores_title"></h2>
             data-ng-bind="$row.size.value + ' ' + ('unit_size_' + $row.size.unit | translate)"
         ></span>
     </oui-datagrid-column>
+    <oui-datagrid-column
+        data-title=":: 'dedicatedCloud_tab_datastores_activeNode' | translate"
```
Comment (anooparveti) [inline packages/manager/apps/dedicated/client/app/components/dedicated-cloud/datacenter/datastore/dedicatedCloud-datacenter-datastore.html:33]:
```suggestion
        data-title=":: 'dedicatedCloud_tab_datastores_active_node' | translate"
```

File: packages/manager/apps/dedicated/client/app/components/dedicated-cloud/datacenter/datastore/dedicatedCloud-datacenter-datastore.html [hunk 2]
```diff
@@ -34,6 +29,17 @@ <h2 data-translate="dedicatedCloud_tab_datastores_title"></h2>
...
+    >
+        <span
+            data-ng-bind="'dedicatedCloud_tab_datastores_activeNode_' + $row.activeNode | translate"
```
Comment (anooparveti) [inline packages/manager/apps/dedicated/client/app/components/dedicated-cloud/datacenter/datastore/dedicatedCloud-datacenter-datastore.html:36]:
```suggestion
            data-ng-bind="'dedicatedCloud_tab_datastores_active_node_' + $row.activeNode | translate"
```

File: packages/manager/apps/dedicated/client/app/components/dedicated-cloud/datacenter/datastore/translations/Messages_fr_FR.json [hunk 3]
```diff
@@ -4,6 +4,9 @@
   "dedicatedCloud_tab_datastores_name": "Nom",
   "dedicatedCloud_tab_datastores_profile": "Profil",
   "dedicatedCloud_tab_datastores_type": "Type",
+  "dedicatedCloud_tab_datastores_activeNode": "Noeud actuel",
+  "dedicatedCloud_tab_datastores_activeNode_master": "Primaire",
+  "dedicatedCloud_tab_datastores_activeNode_slave": "Réplicat",
```
Comment (anooparveti) [inline packages/manager/apps/dedicated/client/app/components/dedicated-cloud/datacenter/datastore/translations/Messages_fr_FR.json:9]:
```suggestion
  "dedicatedCloud_tab_datastores_active_node": "Noeud actuel",
  "dedicatedCloud_tab_datastores_active_node_master": "Primaire",
  "dedicatedCloud_tab_datastores_active_node_slave": "Réplicat",
```

Comment (carsso) [reply packages/manager/apps/dedicated/client/app/components/dedicated-cloud/datacenter/datastore/translations/Messages_fr_FR.json]: done

File: packages/manager/apps/dedicated/client/app/components/dedicated-cloud/datacenter/datastore/dedicatedCloud-datacenter-datastore.html (hunk 2 above)
Comment (carsso) [reply packages/manager/apps/dedicated/client/app/components/dedicated-cloud/datacenter/datastore/dedicatedCloud-datacenter-datastore.html]: done

File: packages/manager/apps/dedicated/client/app/components/dedicated-cloud/datacenter/datastore/dedicatedCloud-datacenter-datastore.html (hunk 1 above)
Comment (carsso) [reply packages/manager/apps/dedicated/client/app/components/dedicated-cloud/datacenter/datastore/dedicatedCloud-datacenter-datastore.html]: done

**Review by dectotam — DISMISSED** [review]

**Review by anooparveti — DISMISSED** [review]

**Review by dectotam — APPROVED** [review]

**Review by sachinrameshn — APPROVED** [review]

**Pull request merged**